lacommunaute/forum_conversation/forms.py

```diff
@@ -5,21 +5,31 @@
 from machina.conf import settings as machina_settings
 from taggit.models import Tag
 
+from lacommunaute.forum_conversation.models import Post
 from lacommunaute.forum_moderation.utils import check_post_approbation
 
 
-class UpdatePostMixin:
+class CreateUpdatePostMixin:
+    def clean(self):
+        cleaned_data = super().clean()
+        if "content" in cleaned_data:
+            post = check_post_approbation(
+                Post(username=cleaned_data.get("username", None), content=cleaned_data.get("content"))
```

Comment:
Si je ne m'abuse, le comportement de `.get` est de retourner `None` par défaut s'il ne trouve pas la clé.

```suggestion
                Post(username=cleaned_data.get("username"), content=cleaned_data.get("content"))
```